Makefile

```diff
@@ -130,7 +130,7 @@ ci:	## Run in automation environment
 	$(eval DISABLE_PASSCODE=true)
 	$(eval AUTO_APPROVE=-auto-approve)
 	$(eval SP_AUTH=true)
-	$(eval CONFIRM_DEPLOY=true)
+	$(eval SKIP_CONFIRM=true)
```

Comment:
This is only half implemented. Please check the template to see how SKIP_CONFIRM is used